apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.ts

```diff
@@ -345,4 +345,20 @@ export class ContentFilterComponent implements DoCheck, OnInit, OnChanges, OnDes
             this.signatureImgAttached = false;
         }
     }
+
+    /**
+     * Change voucher number or date based on Invoice number or date
+     *
+     * @param {boolean} [isDate=true] True, if date is changed
+     * @memberof ContentFilterComponent
```

Comment:
Optimise this code 

public handleInvoiceDateNumberChange(isDate: boolean = true): void {
    const type = isDate ? 'date' : 'number';
    const sourceKey = `invoice${type}`;
    const targetKey = `voucher${type}`;
    
    this.customTemplate.sections['header'].data[targetKey].label = this.customTemplate.sections['header'].data[sourceKey].label;
    this.customTemplate.sections['header'].data[targetKey].display = this.customTemplate.sections['header'].data[sourceKey].display;
}